src/schemas.py

```diff
@@ -6,7 +6,9 @@
 
 # -- UTILITY TYPES --
 class ConditionType(enum.Enum):
+    # Max number of calls to a given entrypoint, for all sponsee
     MAX_CALLS_PER_ENTRYPOINT = "MAX_CALLS_PER_ENTRYPOINT"
+    # Max number of calls per sponsee per contract
```

Comment:
So it's not the same business rule which is mentionned in the PR #45 , right ?

Reply:
Yes it is, but apparently I can't get the wording right. :)

The goal is to limit the subsidy for new users, for the smart contract targeted by the condition. So we want users to be able to submit N operations to this smart contract, but not more. But this should not impact other users, including newer accounts.

I suppose that it could be "max number of calls per sponsee per vault" instead?